What We Do and Get Involved: Geoshapes

sreya-kata opened the pull request:
- Added geoshapes to get-involved page
- Adjusted geoshape imports and positioning for "what we do: platforms" and "get involved: introduction"
- Abstracted geoshape into its own component to be reused
**_- With this PR, the `get-involved` page should be complete and the `what-we-do` page should also be complete with PR #74_** 
![shapes](https://user-images.githubusercontent.com/60420625/115745554-8bf24e00-a361-11eb-8a4d-533494beb278.jpg)

sreya-kata committed:
added images to folder

sreya-kata committed:
added bottom svg image

sreya-kata committed:
added top svg image

sreya-kata committed:
added middle svg shape

sreya-kata committed:
finalized geoshapes and rename them

sreya-kata committed:
organized svgs into folders

sreya-kata committed:
abstracted geoshapes into their own component

sreya-kata committed:
got rid of unused import

sreya-kata committed:
adjusted intro shape for get-invovled

sreya-kata committed:
got rid of unused variables

sreya-kata committed:
adjusted responsiveness of getinvolved intro shape

josephmannis reviewed (commented):
From a code perspective this looks good. I noticed some design inconsistencies so please get approval from Riha first.

On what we do, developed:
![image](https://user-images.githubusercontent.com/29356116/115901078-bcec8480-a42e-11eb-9575-f678bf7eb690.png)

Vs designed:
![image](https://user-images.githubusercontent.com/29356116/115901149-d2fa4500-a42e-11eb-8475-19c5aafd8755.png)

sreya-kata committed:
adjsted minor design stuff of our platforms, need to fix bottom margin

sreya-kata committed:
*think* i finished the weird margin whitespace thing for the svg

sreya-kata committed:
delete irrelevant folder

sreya-kata committed:
adjusted bottom svgs of get-involved

sreya-kata committed:
fixed responsiveness stuff

sreya-kata committed:
Merge branch 'main' of https://github.com/Scout-NU/vital-site into get-involved/geoshapes

sreya-kata committed:
moved desktop responsiveness to another branch

sreya-kata commented:
Checked with Riha and also did the following:
- replaced the "our platforms" svg with one with no whitespace on the bottom, also decreased margin on top
- added a mobile version of an svg for the "application info" section of the "get-involved" page
- adjusted whitespace for "application info" section based on device width
- bug with position cards content being cut off. fixed this and adjusted line height to align with design
- made a separate branch for desktop/large screen (> 1440px) responsiveness. will make a PR for this later
